### PR TITLE
fix: codelens doesn't show for V3, bot template issue

### PR DESCRIPTION
**Pull request by SLdragon**
Fix below issues:

Fix aad codelens doesn't show for v3
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/16740671

Fix aad codelens preview not work issue:
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/16750883

Fix bot sso template issue:
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/16750839
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/16750875

E2E TEST: N/A

### CODE REVIEW

File: packages/fx-core/src/core/FxCoreImplementV3.ts [hunk 1]
```diff
@@ -212,6 +212,43 @@ export class FxCoreV3Implement {
     }
   }
 
+  @hooks([ErrorHandlerMW, ProjectConsolidateMW, EnvLoaderMW(false)])
+  async previewAadManifest(inputs: Inputs): Promise<Result<Void, FxError>> {
```
Comment (xzf0587) [inline packages/fx-core/src/core/FxCoreImplementV3.ts:216]: The methods in fxCoreImplementV3 are the exposed interface implementation from fxCore.
PreviewAadManifest is inside executeUserTask method. 
Remove the middlewares ion it and move it to the end of class which will make the method usage clear

File: packages/fx-core/src/component/driver/aad/interface/updateAadAppArgs.ts [hunk 2]
```diff
@@ -4,4 +4,5 @@
 export interface UpdateAadAppArgs {
   manifestTemplatePath: string; // Relative path to project root
   outputFilePath: string; // Relative path to project root
+  onlyBuild?: boolean; // Only build aad manifest for preview purpose
```
[the remark this answers is not in the document]
Comment (SLdragon) [reply packages/fx-core/src/component/driver/aad/interface/updateAadAppArgs.ts:7]: Sure, will extract the code

File: packages/fx-core/src/core/FxCoreImplementV3.ts (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/src/core/FxCoreImplementV3.ts:216]: Sure

**Review by xzf0587 — APPROVED** [review]

**Review by xzf0587 — DISMISSED** [review]

**Review by yiqing-zhao — DISMISSED** [review]

**Review by wenytang-ms — DISMISSED** [review]

**Review by tecton — DISMISSED** [review]

File: packages/fx-core/src/component/driver/aad/interface/updateAadAppArgs.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (blackchoey) [reply packages/fx-core/src/component/driver/aad/interface/updateAadAppArgs.ts:7]: let's remove the parameter as we don't need it any more

**Review by blackchoey — DISMISSED** [review]

**Review by tecton — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**